### ISSUE TITLE
Logging in/out not working from issue page

### ISSUE THREAD

**Issue by miketaylr** [webcompat/webcompat.com#1384]
See https://github.com/webcompat/web-bugs/issues/4907

STR: go to an issue, hit log in or log out

Expected: should work
Actual: doesn't!

It seems to be fine for me on the homepage, so that's weird.

**Comment by miketaylr** [comment]
https://s:s@staging.webcompat.com/issues/515, for example.

**Comment by miketaylr** [comment]
This is a recent regression -- I tested and confirmed that it works if you check out the `v3.0.0` tag, but is busted in the `v3.1.0` tag.

```
🐓 git log --pretty=oneline v3.0.0...v3.1.0
cb05a28bb8f2a871730c8b587d3a6e4ca0407349 v3.1.0 changelog update
f5a9ec64bb363bd14eb7ea52518ab834b0e53f6e Merge pull request #1367 from webcompat/issues/1329/1
c8065f7b0594e84ed7aa795b5437638c378d1359 Issue #1329. Add ability to disable CSP logging via config.
4f5a7bd9f81b6cd2b78bb91d04b8dff3fa088e2b Merge pull request #1373 from webcompat/issues/1372/1
32cfb943e71608ecd0a5b95734449cc0680e7307 Issue #1372 - change twitter link in footer
c5e6acfef36d110f34da20d3d26516256fcc278b Merge pull request #1365 from webcompat/issues/1364/1
6097758aac59af5b68d1da2ff57a9e99e2dadf49 Issue #1329. Add URL tests for new report endpoint.
a74fbb70d2a72b47f5a4be4fe4205d37668b1613 Issue #1329. Add a Content-Security-Policy-Report-Only header.
fdcacecfa42ab2f4e802dacc4a2d4b429176eea3 Issue #1328. Add security-related headers into @app.after_request.
b6e7456c454d61175a0e46959abd8610857b0539 Issue #1329. Add a /csp-report endpoint for simple logging of violations.
8aefae120cafe613aabe3d84191c08b9c113a9a1 Merge pull request #1335 from karlcow/609/2
3f6fb43229663a9bd2d7a14cd092c4b8ac40794f Merge pull request #1362 from webcompat/issues/1359/1
57c12cea9f978462bd36e736fb38813e2d45cf5a Issue #1364 - removed eslint related tasks
154f4b1adc16f50d71df6ed31e90aba78c55e144 Merge pull request #1366 from webcompat/issues/1363/1
15c4487528269799a155ec76d175c1d02e84930b Issue #1364 - add lint task to travis.yml
3e6b8b2a863c7f4e680aeb27befb7dd2c4ed6e0d Issues #1363 - adding npm run lint/fix to CONTRIBUTE
85b0b891c0dfba399b57770492f93a3e877d40d0 Issue #1364 - add specific paths to js lint
6be0811511c70ae9383850056e4f6c2b63e92857 Issue #1359. Add tests for problem_type param.
ae276a252c5f118cdcade188771f44c6b6d77c3d Issue #1359. Cleanup in reporting-non-auth.js
8d2800c50121eb5d33dac35f296623bcee558864 Issue #1359. Make it possible to select problem type from problem_type param.
bab6f6df1676b65f891ace94527d4596f176dcbf Issue #609 - Implement Cache-Policy decorator
```

So it's either CSP or Cache-Policy that broke this. cc @karlcow 

Will bisect and we should know very soon.

**Comment by miketaylr** [comment]
Yeah, looks like #609 broke this. To verify:

`git checkout bab6f6df1676b65f891ace94527d4596f176dcbf` and try STR.
then `git checkout 815cb730fed698cca5f3adffba245f0b62bb837f` and the bug doesn't reproduce.

I'm inclined to back this out and deploy a new version if there aren't any messy conflicts.

**Comment by miketaylr** [comment]
Fixed by db4e879a167604debfa04d3e208be68cb4f64c6d.

**Comment by karlcow** [comment]
Just for the record and not forgetting the issue. The login/logout **is working**. There is no issue there. The issue is that we don't see the change of state anymore. See my comment: 
https://github.com/webcompat/webcompat.com/issues/609#issuecomment-284286469 and follow-up discussions.
